### ISSUE TITLE
Loss goes to nan during training

### ISSUE THREAD

**Issue by kdhasi** [linjx-ustc1106/TuiGAN-PyTorch#17]
Hi,

I am impressed with the visual results that TuiGAN has demonstrated in the [paper](https://www.ecva.net/papers/eccv_2020/papers_ECCV/papers/123490018.pdf).

However, I'm encountering an issue while training (using the given script) the model on the apple and orange images provided in the repository. The loss values keep increasing with each scale before eventually going to NaN. As a result, the model only generates an image for scale 1 (attached below) while the rest are blank.

![apple_orange_scale1](https://github.com/linjx-ustc1106/TuiGAN-PyTorch/assets/146899852/7bbfacad-9462-4f12-8e56-fc5f079b32d0)

Am I missing something in the overall training procedure? Any guidance or suggestions would be greatly appreciated. Thank you and looking forward to hearing from you.
